### PR TITLE
TLMBLGIS-73 Modify DB to store intention of attending event

### PR DESCRIPTION
**Pull request by dovek**
Due: 7/25
@dlhaines 
@pushyamig

### CODE REVIEW

File: mbof/serializers.py [hunk 1]
```diff
@@ -18,10 +18,16 @@ class EventSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = Event
         fields = ('url', 'title', 'status', 'address', 'eventText', 'latitude', 'longitude', 'altitudeMeters', 'owner', 'postingTime', 'startTime',
-                  'endTime', 'hashtag', 'votes', 'category',)
+                  'endTime', 'hashtag', 'votes', 'guests', 'category',)
```
Comment (dlhaines) [inline mbof/serializers.py:21]: Does 'guests' have anything to do with this functionality?

File: mbof/models.py [hunk 2]
```diff
@@ -89,6 +89,17 @@ def votes(self):
         )
         return voteTotal
 
+    @property
+    def guests(self):
+        guestTotal = reduce(
```
Comment (dlhaines) [inline mbof/models.py:94]: Reduce, lambda are great.
The test looks obscure to me, though it is short enough to be readable as is.  Maybe add intension.count method that returns 0 or 1?

**Comment by dlhaines** [comment]
I also added a comment to the jira.

File: mbof/serializers.py (hunk 1 above)
Comment (pushyamig) [inline mbof/serializers.py:21]: I agree  with dave. how are `guests` and intentions are linked?

Comment (dovek) [inline mbof/serializers.py:21]: When you have an intention to go to an event then you are registered as a guest to that event.

**Pull request merged**